### PR TITLE
Phase1

### PR DESCRIPTION
**Pull request by supremebeing7**
Enables only a handful of cops to begin with, and all are ones that support offense auto-correction.

### CODE REVIEW

File: default.yml [hunk 1]
```diff
@@ -148,207 +149,27 @@ AllCops:
   TargetRubyVersion: ~
   TargetRailsVersion: 5.0
 
-# Indent private/protected/public as deep as method definitions
-Layout/AccessModifierIndentation:
```
Comment (Parmandil) [inline default.yml:152]: This seems like one maybe we could keep

File: default.yml [hunk 2]
```diff
@@ -538,21 +251,56 @@ Layout/EmptyLinesAroundModuleBody:
...
+# Use empty lines between defs.
+Layout/EmptyLineBetweenDefs:
+  # If `true`, this parameter means that single line method definitions don't
+  # need an empty line between them.
+  AllowAdjacentOneLineDefs: false
+  # Can be array to specify minimum and maximum number of empty lines, e.g. [1, 2]
+  NumberOfEmptyLines: 1
+
+Layout/SpaceAroundEqualsInParameterDefault:
+  EnforcedStyle: space
```
Comment (Parmandil) [inline default.yml:263]: This is the only situation with `=` where I kind of prefer `no_space`, but if it's typical to have spaces in the Ruby community then okay.

File: default.yml [hunk 3]
```diff
@@ -768,846 +392,1288 @@ Layout/IndentHash:
...
+#   Enabled: true
+#
+# Performance/HashEachMethods:
+#   Enabled: true
+#
+# Lint/UselessAssignment:
+#   Enabled: true
+#
+# Layout/EmptyLines:
+#   Enabled: true
```
Comment (Parmandil) [inline default.yml:425]: Seems like a strong candidate to me

**Review by Parmandil — APPROVED** [review]

**Comment by Parmandil** [comment]
There's several more I hope we can reenable relatively quickly, but this seems like a huge step in the right direction.

File: default.yml (hunk 1 above)
Comment (supremebeing7) [reply default.yml:152]: For this first phase, I was trying to only do rules that were autocorrect-able, and surprisingly this was not one of them (or if it is, autocorrect was not working well with it at all). But I will move it to Phase 2 Candidates section.

File: default.yml (hunk 2 above)
Comment (supremebeing7) [reply default.yml:263]: Yeah, I think it is, but if you want you can raise the point in slack

File: default.yml (hunk 3 above)
Comment (supremebeing7) [reply default.yml:425]: Another one where autocorrect failed.

**Review by ekluff — APPROVED** [review]

**Pull request merged**